client/src/components/LinkField/LinkField.js

```diff
@@ -19,36 +22,63 @@ const section = 'SilverStripe\\LinkField\\Controllers\\LinkFieldController';
  * onChange - callback function passed from JsonField - used to update the underlying <input> form field
  * types - injected by the GraphQL query
  * actions - object of redux actions
+ * isMulti - whether this field handles multiple links or not
  */
-const LinkField = ({ value, onChange, types, actions }) => {
-  const linkID = value;
-  const [typeKey, setTypeKey] = useState('');
+const LinkField = ({ value = null, onChange, types, actions, isMulti = false  }) => {
   const [data, setData] = useState({});
-  const [editing, setEditing] = useState(false);
+  const [editingID, setEditingID] = useState(null);
```

Comment:
```suggestion
  const [editingID, setEditingID] = useState(-1);
```
Are we using null because 0 is reserved for editing new unsaved Links?

`-1` is better than `null` because it's easy to make errors by testing the truthiness of editingID when 0 is a valid value. Instead explicitly test for `!== -1` similar to `indexOf()`.

Reply:
We're using null because it's unset, and we can easily just do a truthy check against it. If there's an ID, it'll be truthy. I'll change to `0` if you want but `-1` is nonsensical and isn't what we use _anywhere_ to denote "not an ID".

Reply:
OK `0` makes sense

Reply:
Done